packages/common/src/operations.graphql

```diff
@@ -1,54 +1,3 @@
-query names(
```

Comment:
Again a big diff, what was changed here was simply the removal of the `itemsListViw` generated code